wooey/tasks.py

```diff
@@ -234,7 +234,11 @@ def cleanup_dead_jobs():
 
     # Get active tasks from Celery
     inspect = celery_app.control.inspect()
-    active_tasks = {task['id'] for worker, tasks in six.iteritems(inspect.active()) for task in tasks}
+    task_infos = inspect.active()
```

Comment:
Could you call this `worker_info`? I think you can also achieve the None check here by doing:
`worker_info = inspect.active() or {}`